File: Exercícios/Exercicios/src/exercicios_java/Ex26.java
package exercicios_java;

import java.util.Scanner;

public class Ex26 {

	public static void main(String[] args) {
		// Encontrar o dobro de um número caso ele seja positivo e o seu triplo caso seja 
		// negativo, imprimindo o resultado.
		
		Scanner ler = new Scanner (System.in);
		
		int a, res;
		
		System.out.printf("Entre com um valor: ");
		a = ler.nextInt();
		if (a > 0) {
			res = 2 * a;
			System.out.printf("O dobro do valor digitado é: %d",res);
		}else {
			res = 3 * a;
			System.out.printf("O triplo do valor digitado é: %d",res);
		}
		
		
		
		
		
		
		
	}

}

File: Python/Exercicios/Ex26.py
# Dobro do número se for positivo, e triplo do numero se for negativo

a = int(input("Digite um numero: "))

if (a > 0):
    res = a * 2
    print(f"O dobro do numero digitado é {res}")
else :
    res = a * 3
    print(f"O triplo do numero digitado é {res}")